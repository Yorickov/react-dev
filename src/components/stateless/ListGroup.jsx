import React, { Children } from 'react';

const ListGroup = ({ children }) => (
  <ul className="list-group">
    {Children.map(children, (child) => (
      <li className="list-group-item">{child}</li>))}
  </ul>
);

export default ListGroup;
