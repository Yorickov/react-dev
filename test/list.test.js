import React from 'react';
import ListGroup from '../src/components/simple/ListGroup.jsx';

it('ListGroup', () => {
  const wrapper = shallow(
    <ListGroup>
      <p>one</p>
      <p>two</p>
    </ListGroup>,
  );

  expect(wrapper.render()).toMatchSnapshot();
});
