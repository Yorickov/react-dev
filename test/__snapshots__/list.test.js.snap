// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListGroup 1`] = `
<ul
  class="list-group"
>
  <li
    class="list-group-item"
  >
    <p>
      one
    </p>
  </li>
  <li
    class="list-group-item"
  >
    <p>
      two
    </p>
  </li>
</ul>
`;
